common/taskana-common/src/main/resources/sql/db2/taskana_schema_update_5.10.0_to_6.2.0_db2.sql

```diff
@@ -0,0 +1,9 @@
+-- this script updates the TASKANA database schema from version 5.10.0 to version 6.2.0.
```

Comment:
It also does additional stuff like altering/updating a table. Older update scripts mentioned what exactly the script does (which tables were affected), but somehow the newer scripts don't contain that anymore and only state the version upgrade. That's okay, but I don't remember if we explicitly decided to keep this as a one liner